client/components/Content/content.css

```diff
@@ -0,0 +1,32 @@
+@import '../../style/colors';
+@import '../../style/variables';
+@import '../../style/mixins';
+
+:root {
+  --content-padding-left: 20px;
+}
+
+.content {
+  position: relative;
+  display: inline-block;
+  overflow-y: auto;
+  overflow-x: hidden;
+  flex-grow: 1;
+  z-index: 1;
+  -webkit-overflow-scrolling: touch;
+  margin-left: var(--sidebar-width);
+  padding-left: var(--content-padding-left);
+  padding-top: var(--content-padding-left);
+
+  & .print-wrapper {
```

Comment:
doesn't need to be nested for 3 classes. Move to css-modules and remove postcss-nested.